client/src/deploy/send.rs

```diff
@@ -0,0 +1,46 @@
+use clap::{App, ArgMatches, SubCommand};
+
+use casper_node::types::Deploy;
+
+use super::creation_common;
+use crate::{command::ClientCommand, common, RpcClient};
+
+pub struct SendDeploy;
+
+// TODO: rename casper_node account type PutDeploy => SendDeploy, and account_send_deploy
+impl RpcClient for SendDeploy {
+    const RPC_METHOD: &'static str = "account_put_deploy";
+}
+
+impl<'a, 'b> ClientCommand<'a, 'b> for SendDeploy {
+    const NAME: &'static str = "send-deploy";
+    const ABOUT: &'static str =
+        "Deploy a smart contract file to the Casper network via an existing running node.";
+
+    fn build(display_order: usize) -> App<'a, 'b> {
+        let subcommand = SubCommand::with_name(Self::NAME)
+            .about(Self::ABOUT)
+            .display_order(display_order)
+            .arg(creation_common::input::arg(display_order));
+        creation_common::apply_common_creation_options(subcommand)
+    }
+
+    fn run(matches: &ArgMatches<'_>) {
+        creation_common::show_arg_examples_and_exit_if_required(matches);
```

Comment:
I don't think we need this.  This is just to show the complicated input format for passing session or payment args in subcommands which actually construct a new deploy (put-deploy, make-deploy and transfer).

This subcommand should just read in a deploy from file and send it to the node.  There's no deploy creation going on.